src/utils/helpers/translations/mapColumnsByRole.ts

```diff
@@ -11,52 +25,60 @@ import COLUMNS_PER_ROLE from '../../model/columnsPerUserRole'
  * - @returns Object with each Column details to construct header and rows of applications list.s
  */
 
-interface ColumnDetails {
-  headerName: string
-  filters: Array<string>
-  render?: Function
-}
-
-const allColumns = {
+const allColumns: { [key in APPLICATION_COLUMNS]: ColumnDetails } = {
+  SERIAL_NUMBER: {
+    headerName: 'Serial number',
+    filters: ['serial'],
```

Comment:
I'm wondering if these filters need to be defined here. We're going to have to have a mapping method to convert URL query filters to actual GraphQL filters, so they'll all need to be defined elsewhere anyway, and I don't think it matter which columns they go with. What was your thinking for having this `filters` field?

Reply:
I thought that returning which ones are the filters (for each column) in the array of columns
https://github.com/openmsupply/application-manager-web-app/blob/48c8899fe539e62ad8392c552f06784b0aa9a16c/src/utils/helpers/translations/mapColumnsByRole.ts#L81-L83
would be useful to map something that should be displayed in the Cell. For example, using the `search` filter would query for a new list of applications and highlight the **start of words** considered in the query, for specific columns.
I'm happy to remove from there if you think it's not helping if the filtering.

Reply:
I think this will be taken care of with what I'm doing with the filtering handling. But maybe just leave it for now and I'll remove them in my PR once I'm sure they're not needed.